spec/services/session_encryptor_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe SessionEncryptor do
+  describe '#load' do
```

Comment:
Yes, the app should completely break wherever the session is used, if you change the method names.